types/pop.go

```diff
@@ -0,0 +1,35 @@
+package types
+
+import (
+	"errors"
+	"github.com/babylonlabs-io/babylon/crypto/bls12381"
+	"github.com/babylonlabs-io/babylon/x/checkpointing/types"
+	"github.com/cometbft/cometbft/crypto/ed25519"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+)
+
+func validPop(blsPubkey bls12381.PublicKey, valPubkey cryptotypes.PubKey, ed25519Sig, blsSig []byte) bool {
```

Comment:
maybe lets add some doc why we are doing this i.e in the TGE chain we have different format of PoP in genesis and we want to avoid migration and monitor check only genesis